.travis.yml

```diff
@@ -16,8 +16,8 @@
 language: groovy
 
 jdk:
-  - openjdk6
```

Comment:
I hope this will fix the [build](https://travis-ci.org/apache/groovy/jobs/141155140)